packages/internal/dex/sdk/src/types/index.ts

```diff
@@ -6,7 +6,7 @@ export type ExchangeContracts = {
   coreFactory: string;
   quoterV2: string;
   peripheryRouter: string;
-  secondaryFee: string;
```

Comment:
Will this break the overrides in the sample app?

Reply:
GOOD POINT. A job for next year 🖖